parlai/scripts/extract_image_feature.py

```diff
@@ -103,10 +103,13 @@ def extract_feats(opt):
         world = create_task(opt, agent)
 
         total_exs = world.num_examples()
-        # TODO: wrap in a tqdm
+        pbar = tqdm.tqdm(unit='ex', total=total_exs)
         while not world.epoch_done():
             world.parley()
+            pbar.update()
```

Comment:
this idiom appears a bunch of places. is there a better improvement? no action required here but